From: Sabyrzhan Tasbolatov <snovitoll@gmail.com>
Subject: Documentation/kasan: fix indentation in translation
Date: Sun, 20 Oct 2024 09:28:13 +0500

Fix the warning in linux-next (htmldocs):

> Documentation/translations/zh_TW/dev-tools/kasan.rst:410:
>	ERROR: Unexpected indentation.

This is based on -mm tree (linux-mm-unstable branch).

Link: https://lkml.kernel.org/r/20241020042813.3223449-1-snovitoll@gmail.com
Signed-off-by: Sabyrzhan Tasbolatov <snovitoll@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/translations/zh_TW/dev-tools/kasan.rst |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Documentation/translations/zh_TW/dev-tools/kasan.rst~kasan-delete-config_kasan_module_test-fix
+++ a/Documentation/translations/zh_TW/dev-tools/kasan.rst
@@ -407,7 +407,7 @@ 有一些KASAN測試可以驗證KASAN是
 
 所有 KASAN 測試均與 KUnit 測試框架集成，並且可以啟用
 透過 ``CONFIG_KASAN_KUNIT_TEST``。可以運行測試並進行部分驗證
- 以幾種不同的方式自動進行；請參閱下面的說明。
+以幾種不同的方式自動進行；請參閱下面的說明。
 
 如果偵測到錯誤，每個 KASAN 測試都會列印多個 KASAN 報告之一。
 然後測試列印其編號和狀態。
_
